src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -672,6 +672,7 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                         ->scalarNode('version')->defaultNull()->end()
                         ->scalarNode('version_format')->defaultValue('%%s?%%s')->end()
                         ->scalarNode('json_manifest_path')->defaultNull()->end()
+                        ->booleanNode('json_manifest_allow_missing')->defaultFalse()->end()
```

Comment:
what about `->defaultValue(!$this->debug)`?

Reply:
A default logic should not depend on any logic, it should be the default, but could be set in the config via “kernel.debug” parameter via the receipe IMO

Reply:
I guess we should keep the "before" behaviour by passing false by default.